### PR TITLE
Use regular font-size and styling in uploader

### PR DESCRIPTION
**Pull request by guerler**
In some cases the font-sizes used in the upload view are too small compared to the rest of the UI. Additionally there are some inconsistent element dimensions. This PR increases the font-sizes to regular. It aligns the style of the upload view, reduces style customizations and refactors the template strings using ES6 literals.

untitled

### CODE REVIEW

**Comment by dannon** [comment]
Here are the two side by side for comparison.

before:
image

after:
image

File: client/galaxy/scripts/mvc/upload/collection/collection-row.js [hunk 1]
```diff
@@ -160,7 +160,7 @@ export default Backbone.View.extend({
...
+        }" class="upload-row"><td><div class="upload-text-column"><div class="upload-mode"/><div class="upload-title-extended"/><div class="upload-text"><div class="upload-text-info">Download data from the web by entering a URL (one per line) or directly paste content.</div><textarea class="upload-text-content form-control"/></div></div></td><td><div class="upload-size"/></td><td><div class="upload-info"><div class="upload-info-text"/><div class="upload-info-progress progress"><div class="upload-progress-bar progress-bar progress-bar-success"/><div class="upload-percentage">0%</div></div></div></td><td><div class="upload-symbol ${
```
Comment (nsoranzo) [inline client/galaxy/scripts/mvc/upload/collection/collection-row.js]:
```suggestion
        }" class="upload-row"><td><div class="upload-text-column"><div class="upload-mode"/><div class="upload-title-extended"/><div class="upload-text"><div class="upload-text-info">Download data from the web by entering URLs (one per line) or directly paste content.</div><textarea class="upload-text-content form-control"/></div></div></td><td><div class="upload-size"/></td><td><div class="upload-info"><div class="upload-info-text"/><div class="upload-info-progress progress"><div class="upload-progress-bar progress-bar progress-bar-success"/><div class="upload-percentage">0%</div></div></div></td><td><div class="upload-symbol ${
```

File: client/galaxy/scripts/mvc/upload/default/default-row.js [hunk 2]
```diff
@@ -280,7 +280,7 @@ export default Backbone.View.extend({
                             <input class="upload-title ml-2 border rounded"/>
                             <div class="upload-text">
                                 <div class="upload-text-info">
-                                    You can tell Galaxy to download data from web by entering URL in this box (one per line). You can also directly paste the contents of a file.
+                                    Download data from the web by entering a URL (one per line) or directly paste content.
```
Comment (nsoranzo) [inline client/galaxy/scripts/mvc/upload/default/default-row.js]:
```suggestion
                                    Download data from the web by entering URLs (one per line) or directly paste content.
```

**Comment by guerler** [comment]
Thanks @nsoranzo.

**Comment by dannon** [comment]
@guerler I merged nsoranzo's suggestions in because I'd planned on merging the whole thing, but I'm tinkering with some padding options for `.upload-top-info` prior to that.  The text seems close to the line, but I don't like how options I'm coming up with work with the whitespace when there's nothing there, so far.

**Comment by dannon** [comment]
@guerler Finally got back to tweaking the text, look good to you?

image

(so it's all vertically centered with flexible padding that should scale with text size changes)

**Comment by guerler** [comment]
Looks good. Thanks!

**Comment by dannon** [comment]
@guerler Cool.  Thanks for the fixes!

**Pull request merged**